app/windows/Import/Components/Form.jsx

```diff
@@ -20,7 +20,8 @@ class Form extends React.Component {
     if (isValid) {
       this.props.statsStore.hash = hash
       this.props.statsStore.isValid = isValid
-      this.forceUpdate()
+      this.props.statsStore.check()
+        .then(() => { this.forceUpdate() })
```

Comment:
Should we add a catch here?

Reply:
The catch is in the stats store, shows an error dialog